### ISSUE TITLE
Broken simlinks on Windows - need for system privilege check?

### ISSUE THREAD

**Issue by chadbrewbaker** [uutils/coreutils#3461]
The Windows build is breaking on systems with [default permissions](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/create-symbolic-links) that do not allow creation of symlinks.

Should we add a system privilege check at the start of the build or otherwise segregate features by system privilidge?

**Comment by oech3** [comment]
Hardlink is default on Windows now. Would you close this issue?

**Comment by oech3** [comment]
@cakebaker OK to close this as GNU make creates hardlinks?

**Comment by sylvestre** [comment]
@oech3 unrelated but would you like to get issue triage permissions ? :)

**Comment by sylvestre** [comment]
same question for you @xtqqczze :)

**Comment by oech3** [comment]
> would you like to get issue triage permissions ?

maybe, yes.

**Comment by xtqqczze** [comment]
> same question for you [@xtqqczze](https://github.com/xtqqczze) :)

Yeah, I open quite a few issues, e.g. https://github.com/uutils/coreutils/issues/14339, so having issue triage permissions would definitely be helpful.
